RCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/compute_reservoir_temperature.py

```diff
@@ -0,0 +1,74 @@
+# RCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/compute_reservoir_temperature.py
+#
+#
+# Created:  Apr 2024, S. Shekar 
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  Compute heat loss to environment 
+# ----------------------------------------------------------------------------------------------------------------------
+def compute_reservoir_temperature(RES,state,coolant_line,dt,i):
+    """
+     Computes the resultant temperature of the reservoir at each time step with coolant exchanging heat to the environment
+          
+          Inputs: 
```

Comment:
function header. I would actually use reservoir instead of RES,  or even res instead of RES. Dont think we should use all caps

Reply:
Changes all RES to reservoir